src/main/kotlin/com/cjbooms/fabrikt/generators/client/OkHttpSimpleClientGenerator.kt

```diff
@@ -209,7 +209,7 @@ data class SimpleClientOperationStatement(
             "PATCH" -> this.addRequestSerializerStatement("patch")
             "HEAD" -> this.add("\n.head()")
             "GET" -> this.add("\n.get()")
-            "DELETE" -> this.add("\n.delete()")
+            "DELETE" -> this.addRequestSerializerStatement("delete")
```

Comment:
I haven't cloned the repo yet, but on first light, I think something like this would be a more cohesive solution.

```
"DELETE" -> {
  val hasBodyParameters = parameters.any { it is BodyParameter }
  if (hasBodyParameters) {
      this.addRequestSerializerStatement("delete")
  } else {
       this.add("\n.delete()")
  }
}
```